device/rsvp/rsvpInterface.yaml

```diff
@@ -0,0 +1,79 @@
+components:
+  schemas:
+    Rsvp.Interface:
+      x-status: under-review
+      description: >-
+        Configuration for RSVP Interface.
+      type: object
+      required: [ipv4_name,neighbor_ip]
+      properties:
+        ipv4_name: 
+          description: >-
+            The unique name of the IPv4 interface connected to the DUT.
+            Single hop RSVP messages like RESV,Bundle,Hello and SRefresh are transmitted on this interface.
+          type: string
+          x-constraint:
+          - '/components/schemas/Device.Ipv4/properties/name'
+        neighbor_ip:
```

Comment:
neighbor_ip -> neighbor_ipv4_address

Reply:
no, will keep as neighbor_ip, if rsvpv6 comes then will add Device.Rsvpv6 in that required will have ipv6_name and neighbor_ip, neighbor_ip description will be ipv6 here